libs/form-component/src/styles/global.css

```
@@ -0,0 +1,178 @@
+  overflow: hidden;
+  text-indent: 1.3em;
+}
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)